bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java

```diff
@@ -362,7 +364,9 @@ private void actionOpenClose(String action, OnOffType controlState) {
             if (activeOptions.containsKey(action)) {
                 if (vehicleStatus.carLocked != controlState) {
                     try {
-                        bridgeHandler.postURL(SERVICE_URL + "vehicles/" + configuration.vin + "/" + action, null);
+                        String address = SERVICE_URL + "vehicles/" + configuration.vin + "/" + action;
+                        String json = new String("{}");
+                        bridgeHandler.postURL(address, json);
```

Comment:
Just use `"{}"` the `new String` is redundant and not used this way in java.

Reply:
Corrected with new commit.